docs/source/io.rst

```diff
@@ -17,6 +17,45 @@ Video
 .. autofunction:: write_video
 
 
+Low-level video API
```

Comment:
Do you think we should call it low-level? For me it's more like a fine-grained API, although low-level could be ok, it might also look like we wouldn't encourage using it.